### PR TITLE
Latency tests: remove a condition which is always false

### PR DESCRIPTION
**Pull request by Tal-or**
**Why this fix is needed:**
An issue was discovered during a local build of the cnf-test image that @shajmakh did.
She built the image from https://github.com/Tal-or/cnf-features-deploy/tree/ship_hwlatdetect_cyclictest and ran the latency test which failed with ```"testName variable has incorrect value..."```
a careful examination of the code shows that this check will always fail, IOW the condition will always be false and it will throw this message and exit the test. 

**How do we fix it**
Removing the faulty check. In the first place, this check was intended to verify some flow which eventually was not written at all.

Signed-off-by: Talor Itzhak <titzhak@redhat.com>

### CODE REVIEW

**Comment by cynepco3hahue** [comment]
@Tal-or Why does it WIP PR?

**Comment by Tal-or** [comment]
> @Tal-or Why does it WIP PR?

I wanted to test the fix first

**Comment by fromanirh** [comment]
please explain a bit more in the commit message (and then in the PR description)
1. how you find this
2. how this can help
TL;DR: we need the "why"

**Comment by Tal-or** [comment]
> please explain a bit more in the commit message (and then in the PR description)
> 
> 1. how you find this
> 2. how this can help
>    TL;DR: we need the "why"

Done
